CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -330,6 +333,8 @@ public function postProcess() {
    *
    */
   public function submit($submittedValues) {
+    $this->_params = $submittedValues;
```

Comment:
these fixes are part of the test pr - that PR could be merged separately as it stands alone & is from Pradeeps other pr